### PR TITLE
Implement Merge Sort

### LINKED ISSUE #13: Implement Merge Sort
Ivan needs to write merge sort for this repo.

### PR DESCRIPTION
**Pull request by IvanIsCoding**
Closes #13

### CODE REVIEW

File: merge_sort.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from heapq import merge
+
+def merge_sort(arr):
+
+	N = len(arr)
+	if N <= 1:
+		return arr
+
+	# Recursively sort the array
```
Comment (QuinnMarshal) [inline merge_sort.py:9]: Thanks for the helpful explanation of your code!

**Review by QuinnMarshal — APPROVED** [review]
LGTM!

File: merge_sort.py (hunk 1 above)
Comment (IvanIsCoding) [reply merge_sort.py:9]: It's trivial, you're welcome

**Review by IvanIsCoding — COMMENTED** [review]
Merging

**Pull request merged**